nni/algorithms/hpo/dngo_tuner.py

```diff
@@ -0,0 +1,117 @@
+import logging
+
+import numpy as np
+import torch
+import nni.parameter_expressions as parameter_expressions
+from nni import ClassArgsValidator
+from nni.tuner import Tuner
+from pybnn import DNGO
+from torch.distributions import Normal
+
+_logger = logging.getLogger(__name__)
+
+
+def _random_config(search_space, random_state):
+    chosen_arch = {}
```

Comment:
this tuner is not specific for nas, so better to change "arch" to "config"

Reply:
missed that one. sorry.